src/Parser/RetributionPaladin/Modules/Items/WhisperOfTheNathrezim.js

```diff
@@ -35,14 +37,25 @@ class WhisperOfTheNathrezim extends Module {
     return {
       item: ITEMS.WHISPER_OF_THE_NATHREZIM,
       result: (<dfn data-tip={`
-				The effective damage contributed by Whisper of the Nathrezim.<br/>
+        The effective damage contributed by Whisper of the Nathrezim.<br/>
 				Damage: ${this.owner.formatItemDamageDone(this.damageDone)}<br/>
```

Comment:
This is a little bit redundant since you're showing that as the item's main text.

Reply:
So here are some examples where I want the cloak suggestion to show up

https://www.warcraftlogs.com/reports/RY9yFXkVpZqtzT1B#fight=2&type=damage-done
![99](https://user-images.githubusercontent.com/19510201/30922516-ff71ff2a-a36e-11e7-980d-df61dc39801b.jpeg)

https://www.warcraftlogs.com/reports/aLKgjkyQNWbc2C91#fight=2&type=damage-done
![untitled](https://user-images.githubusercontent.com/19510201/30922863-fec7c126-a36f-11e7-881d-39cd87040b0d.jpeg)


https://www.warcraftlogs.com/reports/BMxLfbYy12pADGCN#fight=1&type=damage-done
![untitled](https://user-images.githubusercontent.com/19510201/30922682-7166f2a2-a36f-11e7-8c3e-b1541a9c667c.jpeg)



Obviously these aren't the only things contributing to these lower than orange parse but they are part of it.